Dockerfile

```diff
@@ -14,13 +14,14 @@ ARG TERRAFORM_VERSION=1.4.5-*
 
 WORKDIR /build
 
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
This was done to resolve this hadolint warning: https://github.com/hadolint/hadolint/wiki/DL4006
Basically to fail/exit if any part of a pipe command fails.

This matters when we download the hashicorp key with this line
```
RUN curl -fsSL https://apt.releases.hashicorp.com/gpg | apt-key add - \
```
The curl could return a 404 and the apt-key command will still try to add that. With this `SHELL` line in place, as as soon as the 404 is received (curl returns exit code 22) the command would fail and the next part of the pipe command would not run.